docs/guide/docker.md

```diff
@@ -0,0 +1,114 @@
+---
+id: docker
+title: 🚢 Build docker image
+description: Build simple docker image that you can use on the production
+sidebar_position: 8
+---
+
+After we have completed the development of our application, the moment comes when we want to upload it to the real world, most often we use Docker for this, wrapping our application in a container.
+
+To do this we need to follow a few simple steps:
+
+## Step 1 - Setup project
+
+First create folder for our project run the following command in the console:
+
+```bash title="~/"
+mkdir example && cd example
+```
+
+Then we have to init go project:
+```bash title="~/example"
+go mod init example.org/demo
+```
+
+Add **Fiber** to our project:
+```bash title="~/example"
+go get github.com/gofiber/fiber/v2
+```
+
+## Step 2 - Hello world
+
+To start we shold create `main.go` file:
+```bash title="~/example"
+touch main.go
+```
+
+Next we should add following content to our file:
+
+```go title="~/app/main.go"
+package main
+
+import "github.com/gofiber/fiber/v2"
+
+func main() {
+	app := fiber.New()
+
+	app.Get("/", func(c *fiber.Ctx) error {
+		return c.SendString("Hello, World!")
+	})
+
+	app.Listen(":4000")
```

Comment:
Update this to `:3000` which the typical one used for expressjs apps

Reply:
Yes, but some of frontent frameworks ( as example react ) also using this port, thats why I usually use `4000`

Reply:
but I will change it to `3000`